app.config.ts

```diff
@@ -1,79 +1,120 @@
 import { ConfigContext, ExpoConfig } from "expo/config";
-import warnOnce from "warn-once";
-// Removed this import as it was causing a build error
-// import type { PluginConfigTypeAndroid } from "expo-build-properties/src/pluginConfig";
+import appConfig from "./versions.json";
 
-import appBuildNumbers from "./app.json";
+const expoEnv = process.env.EXPO_ENV as Environment;
```

Comment:
Todo for later: zod parse and verify configuration shape and provide helpful descriptions of what everything is